translations/round_structure.tex/ua.po

```diff
@@ -101,8 +101,26 @@ msgstr ""
 
 #. type: multicols*
 #: sections/round_structure.tex:30
+#, fuzzy, no-wrap
+#| msgid ""
+#| "{\\centering\\phantom{Star Axis}\\par}\n"
+#| "\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
+#| "\n"
+msgid ""
+"\\includegraphics[width=1.6\\linewidth]{\\art/minotaur.png}\n"
+"\n"
+msgstr ""
+"{\\centering\\phantom{Зоряна Вісь}\\par}\n"
+"\\includegraphics[width=\\linewidth]{\\art/unicorn.jpg}\n"
+"\n"
```

Comment:
One last time https://github.com/Heegu-sama/Homm3BG/pull/580/files#r1640231115

Reply:
Don't worry about them, they are all fuzzy. Discussed in discord